frontend/packages/rhoas-plugin/src/topology/components/rhoasComponentFactory.ts

```diff
@@ -5,6 +5,7 @@ import {
   withSelection,
   withCreateConnector,
 } from '@patternfly/react-topology';
+import BindableNode from '@console/dev-console/src/components/topology/components/BindableNode';
```

Comment:
Should this be part of topology itself if its being used in multiple places? We don't want to create a dependency on dev-console. Topology is going to be a core package and contribute utils and components to the SDK.